src/main/java/git/Ui.java

```diff
@@ -77,20 +77,24 @@ public void displayHelp() {
      * Prints output after setting the selected grocery's expiration date.
      */
     public static void printExpSet(Grocery grocery) {
+        assert !(grocery.getName().isEmpty()): "grocery name should not be empty";
```

Comment:
Good assertions, reminds the users that the grocery name and expiration dates should not be empty